From 9ad2eeb55f61df455d26170f9792f536612929c2 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Wed, 14 Dec 2016 16:26:39 -0800
Subject: [PATCH] IMGAPI-606 imgapi-cli commit 2aef5b0 broke feature that
 'identity' can be a *path to a key file* Reviewed by: Alex Wilson
 <alex.wilson@joyent.com> Approved by: Alex Wilson <alex.wilson@joyent.com>

---
 CHANGES.md   | 6 ++++++
 package.json | 4 ++--
 2 files changed, 8 insertions(+), 2 deletions(-)

diff --git a/CHANGES.md b/CHANGES.md
index e4489d4..fca1665 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -2,6 +2,12 @@
 
 ## not yet released
 
+## 2.2.4
+
+- IMGAPI-606 Fix '--identity' (and the equiv `*_IMGADM_IDENTITY` env var) to
+  once again support taking a path to an SSH private key. This was broken
+  in version 2.0.0.
+
 ## 2.2.3
 
 - TOOLS-1636 Fix updates-imgadm et al to flush stdout on exit.
diff --git a/package.json b/package.json
index cb972e4..0373f43 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "imgapi-cli",
   "description": "a CLI for an Triton's IMGAPI (https://images.joyent.com/docs/)",
-  "version": "2.2.3",
+  "version": "2.2.4",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
@@ -11,7 +11,7 @@
     "mkdirp": "0.5.1",
     "nopt": "2.0.0",
     "progbar": "1.1.1",
-    "sdc-clients": "10.0.3",
+    "sdc-clients": "10.0.4",
     "strsplit": "1.0.0",
     "tabula": "1.8.0",
     "vasync": "1.6.4",
-- 
2.21.0

